### PR TITLE
Table: fix silent filter-change event while calling clearFilter method

### PR DESCRIPTION
**Pull request by EdgarAllanzp**
Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's contributing guide ([中文](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.zh-CN.md) | [English](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.en-US.md) | [Español](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer relative issues for you PR.

Closes https://github.com/ElemeFE/element/issues/15021

### CODE REVIEW

**Comment by ziyoung** [comment]
过滤器清除后，再次点击”清除“，仍然会触发 filterChange。这种改动感觉不太合理。

File: packages/table/src/table-store.js [hunk 1]
```diff
@@ -555,8 +554,8 @@ TableStore.prototype.clearFilter = function(columnKeys) {
     });
     this.commit('filterChange', {
       column: columns,
-      value: [],
-      silent: true,
+      values: [],
+      silent: false,
```
Comment (ziyoung) [inline packages/table/src/table-store.js]: 这里的 silent 应该是计算得来。直接改为 false 不太合理吧。

Comment (EdgarAllanzp) [reply packages/table/src/table-store.js]: https://github.com/ElemeFE/element/pull/15065

我重新提交了，但是貌似带上了其他人的提交

**Comment by ziyoung** [comment]
@EdgarAllanzp 代码冲突了。解决一下冲突。

**Pull request closed without merging**